### PR TITLE
79244 footer placeholder

### PR DESCRIPTION
**Pull request by emily-gullo**
## Summary

- Creating static footer placeholder utilizing the same file format structure as the new `header` for consistency
- Note that links are not yet available for the first column.

## Related issue(s)

- [Ticket #79244](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/79244)

## Testing done

- I ran content-build locally with the new footer to confirm it matches design's mockups
- Further confirmed it is matching the current double-footer state that header is experiencing

## Screenshots

Screenshot 2024-04-08 at 15 15 38

Screenshot 2024-04-12 at 13 02 25

## What areas of the site does it impact?
Accredited Representative Portal

*(Describe what parts of the site are impacted **if** code touched other areas)*

## Acceptance criteria
- [ ] Footer is displaying
- [ ] Footer matches [mocks](https://www.figma.com/file/C3sdnDf8anVqXVdmiCWwq8/%F0%9F%9A%A7-Accredited-Rep-Facing?type=design&node-id=437-34556&mode=design&t=ddYVRQXLxcAZPnWN-0)

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by cohnjesse — DISMISSED** [review]
LGTM, obviously we will need to do a subsequent ticket to remove the content-build footer so we don't have the duplicate. Looks great!

**Review by williamphelps13 — CHANGES_REQUESTED** [review]
Did a first run through! Were you able to run this past design/research to get feedback on this implementation?

File: src/applications/accredited-representative-portal/components/common/Footer/MobileFooter/MobileFooter.jsx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+                For accredited representatives
+              </button>
+              <div
+                className="usa-accordion-content va-footer-accordion-content"
+                aria-hidden="true"
+                id="footer-accredited-reps"
+              >
+                <ul className="va-footer-links">
+                  <li>
+                    <a href="https://va.gov">VA forms</a>
```
[the remark this answers is not in the document]
Comment (emily-gullo) [reply src/applications/accredited-representative-portal/components/common/Footer/MobileFooter/MobileFooter.jsx:45]: I think that'd be ideal for the tickets expanding on this placeholder for sure!

**Review by powellkerry — DISMISSED** [review]

**Review by williamphelps13 — CHANGES_REQUESTED** [review]

**Review by williamphelps13 — APPROVED** [review]
Changes to comments look good. We can address this comment in the next Footer issue: https://github.com/department-of-veterans-affairs/vets-website/pull/29171#discussion_r1576409607

**Pull request merged**